docker-compose.yml

```
@@ -6,7 +6,7 @@ redis:
   env_file: .env
   container_name: denguetorpedo-redis
   volumes:
-    - <local_folder_for_redis>:/bitnami
+    - /home/fernando/Documentos/SE2/denguechat-compose/redis/redisVolume:/bitnami
```

Do not include local paths or info. Keep it general and just indicate what has to be changed/replaced, and how. 
```suggestion
    - <local_folder_for_reds>:/bitnami
```